WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -23709,7 +23709,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = com.automattic.jetpack;
 				PRODUCT_MODULE_NAME = WordPress;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE_SPECIFIER = "WPiOS Development Profile";
+				PROVISIONING_PROFILE_SPECIFIER = "Jetpack iOS Development";
```

Comment:
This one may stand out as out of place, but it's part of #16790. What it does is setting the Jetpack app to use its development provisioning profile, which is appropriate.

I'm not sure what to make of the `PRODUCT_MODULE_NAME = WordPress;` in the Jetpack target but that is out of scope for this PR...

Reply:
I think it's ok to keep the module name as `WordPress`, however odd it might look, because that's for use in symbol namespacing in the code, and if some code in the codebase uses `WordPress.SomeClass.someStaticSymbol` for example to disambiguate, we wouldn't want that code to fail when it's build by the Jetpack target just because of this.

Not sure we have any such disambiguation usage in the code tho, and if not, it could be acceptable and nice for consistency to change it to `Jetpack` for the jetpack target 🤷  (I _do_ like consistency…)

Reply:
I think this change may have been necessary for testing - honestly not sure.  The development profile being specific to Jetpack is in line with all our apps and libs, though, so I think this is fine.

Regarding the module name, I'd avoid making the change here if we don't need to.